src/API/WP/NotificationsService.php

```diff
@@ -186,4 +205,30 @@ public function is_ready( bool $with_health_check = true ): bool {
 	public function is_enabled(): bool {
 		return apply_filters( 'woocommerce_gla_notifications_enabled', true );
 	}
+
+	/**
+	 * Get the DataType from a Notification Topic
+	 *
+	 * @param string $topic The topic.
+	 * @return string The DataType
+	 */
+	protected function get_datatype_from_topic( string $topic ): string {
+		if ( in_array( $topic, self::PRODUCT_TOPICS, true ) ) {
+			return self::get_products_datatype();
```

Comment:
Same here we should use `$this->` instead of `self::`

Reply:
Thanks. Fixed here https://github.com/woocommerce/google-listings-and-ads/commit/e595f4c28c4832b84f9f0079e3c9554aeac82124